### PR TITLE
LOOP-1099: Promote AlertSoundPlayer to LoopKit

### PR DESCRIPTION
**Pull request by rickpasetto**
...that way, it can be used elsewhere (see tidepool-org/dexcom-icgm-loop-plugin#77 for where) for playing alert sounds (for example, in UI for choosing the sound itself).

See also https://github.com/tidepool-org/Loop/pull/74

LW PR: https://github.com/tidepool-org/LoopWorkspace/pull/77

### CODE REVIEW

**Review by nhamming — APPROVED** [review]
LGTM

File: LoopKit/DeviceManager/DeviceAlertSoundPlayer.swift [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        #else
+        AudioServicesPlayAlertSound(SystemSoundID(kSystemSoundID_Vibrate))
+        #endif
+    }
+    
+    public func play(url: URL) {
+        DispatchQueue.main.async {
+            do {
+                // The AVAudioPlayer has to remain around until the sound completes playing.  A cleaner way might be
+                // to wait until that completes, then delete it, but seems overkill.
```
Comment (nhamming) [inline LoopKit/DeviceManager/DeviceAlertSoundPlayer.swift]: [nit] is this comment still valid?

**Review by nhamming — APPROVED** [review]
LGTM

File: LoopKit/DeviceManager/DeviceAlertSoundPlayer.swift (hunk 1 above)
Comment (rickpasetto) [reply LoopKit/DeviceManager/DeviceAlertSoundPlayer.swift]: Aha!  No, it isn't...because I fixed it (it does make the sounds work better).  Good eye.

**Pull request merged**